src/Tizen.NUI.Components/Controls/Slider.Internal.cs

```diff
@@ -623,6 +623,51 @@ private int BgTrackLength()
             return bgTrackLength;
         }
 
+        private int BgTrackOffset()
```

Comment:
method name would be better to have verb such as GetBgTrackOffset()

Reply:
If than, I'd better change the name of `BgTrackLength` also. I'll try to clean up it.

Reply:
done.